network-api/networkapi/wagtailpages/pagemodels/publications/article.py

```diff
@@ -0,0 +1,49 @@
+
+from django.db import models
+
+from wagtail.core.models import Page
+from wagtail.core import blocks
+from wagtail.core.fields import StreamField
+from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
+from wagtail.contrib.table_block.blocks import TableBlock
+
+from ..mixin.foundation_metadata import FoundationMetadataPageMixin
+
+"""
+TODO: 
+agree on featureset for content
+callout may have different featureset, but we mainly want the ability to distinguish it for styling? 
```

Comment:
@phildexter @sabrinang 

So during our stakeholder meeting someone mentioned wanting to be able to put links in callout boxes. 
My instinct was then to create a richtext block that would allow for that sort of thing in a callout box. 
However, as I thought about it, the reason she said she wanted that is so that they could put things link links to feedback forms and such on the page. 
To me that would merit a call-to-action sort of block for that, and not squeeze/hack a call to action inside a component not meant for it. Feels pretty anti 'Zen of Wagtail'. 
So decided to implement the callout box as a simple block quote, and figure we could add an additional feature in order to have survey links, call to actions, or whatever else she was imagining being able to have on the page (which we would probably want a design for). 

Does that make sense? If so we can resolve this code comment and consider that user need for a call to action block a separate issue.